assets/component-cart-notification.css

```diff
@@ -58,8 +58,8 @@
 
 .cart-notification__heading .icon-checkmark {
```

Comment:
Not related to this PR but I am seeing this and not on `main` 🤔 
Are you seeing this as well?

https://screenshot.click/05-20-zzfrg-7d2il.mp4